Mathlib/Algebra/Order/Group/PosPart.lean

```diff
@@ -44,33 +44,9 @@ lattice ordered group.
 positive part, negative part
 -/
 
-/-- The positive part of an element admitting a decomposition into positive and negative parts.
--/
-class PosPart (α : Type*) where
-  /-- The positive part function. -/
-  pos : α → α
-
-#align has_pos_part PosPart
-
-/-- The negative part of an element admitting a decomposition into positive and negative parts.
--/
-class NegPart (α : Type*) where
-  /-- The negative part function. -/
-  neg : α → α
-
-#align has_neg_part NegPart
-
-@[inherit_doc]
-postfix:max "⁺" => PosPart.pos
-
-@[inherit_doc]
-postfix:max "⁻" => NegPart.neg
-
```

Comment:
Did you intentionally remove those? If so, please update the commit message or PR title to reflect that

Reply:
They were supposed to be removed in #9740. I'll mention in the PR description.

Reply:
Can you split this removal to a new PR? I think this one is doing too many things.

Reply:
#10690